src/app/(sidebar)/transaction/submit/components/ErrorResponse.tsx

```diff
@@ -110,6 +110,14 @@ export const RpcErrorResponse = ({ error }: { error: SubmitRpcError }) => {
             />
           </Box>
         ) : null}
+        {diagnosticEvents ? (
```

Comment:
I think we should provide:
- error code
- raw XDR
- diagnostic events (added here)
- a section for decoded XDR to JSON (already have)